packages/backend/src/auth/schemas/signupSchema.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod'
+
+export const signupSchema = z.object({
+  email: z.string().email(),
+  password: z.string()
+})
```

Comment:
```suggestion
import { z } from 'zod'
import { loginSchema } './loginSchema'

export const signupSchema = loginSchema.merge(z.object({
  confirmPassword: z.string()
}))
```

A password confirmation is needed for registration:
https://github.com/interledger/testnet/blob/08b8ed175f0c6b5567bc6b0462f1d01d01b8577d/packages/wallet/src/pages/auth/register.tsx#L10-L26

Reply:
Zod is cool!